### DISCUSSION TITLE
建议增加每个句子之间的停顿时间

### DISCUSSION THREAD

**Discussion by 961882** [clipturbo/clipturbo#9] [Ideas]
建议增加每个句子之间的停顿时间，比如：少儿英语晨读，读单词、句子之间可以调整停顿时间，1-5s。

**Comment by KeJunMao** [comment]
emmm，现在太快了吗

**Reply by 961882** [reply]
1、语速可以0.7。
2、每个句子之间停顿建议可以调节。1-5秒；可以做成听写用的视频。
3、加油；

**Reply by KeJunMao** [reply]
1. 语速现在可以调（部分模型不支持）
2. 我来排个期
3. 感谢！笔芯

**Reply by KeJunMao** [reply]
请教，你希望的停顿是英文和中文之间的还是 相邻的【中文+英文】之间的呢？

**Reply by 961882** [reply]
我理解可以在英文设置一个停顿的毫秒，中文设置一个停顿的毫秒；这样是否会更灵活一些？
